docs/userguides/deactivatedevices.md

```diff
@@ -0,0 +1,65 @@
+# Clean up your environment by deactivating devices
+
+Your Code42 environment may contain many old devices that are no longer active computers, and that have not connected to Code42 in quite some time. In order to clean up your environment, you can use the CLI to deactivate these devices in bulk.
```

Comment:
The comma is not needed after `computers`

Reply:
updated